block/src/Pos/Block/Network/Logic.hs

```diff
@@ -561,6 +563,11 @@ applyWithRollback nodeId enqueue toApply lca toRollback = do
     reportRollback = do
         let rollbackDepth = length toRollback
         let isCritical = rollbackDepth >= criticalForkThreshold
+
+        -- Commit rollback value to EKG
+        whenJustM (view misbehaviorMetrics) $ liftIO .
+            flip Metrics.set (fromIntegral rollbackDepth) . _mmRollbacks
```

Comment:
But wait, the issue description suggests that `_mmRollbacks` gauge corresponds to
> the maximal rollback depth in last T (where is configurable time value, e. g. few hours).

Reply:
We decided that we don't need it and this logic is ok.